Makefile

```diff
@@ -9,13 +9,14 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 	targets/openshift/crd-schema-gen.mk \
 )
 
+IMAGE_TAG?=registry.svc.ci.openshift.org/ocp/4.2:cluster-config-operator
```

Comment:
`IMAGE_TAG` might be confusing, since this is rather pull spec, also I'm not sure we want to allow redlining this other than just the repository part like in https://github.com/openshift/cluster-kube-scheduler-operator/tree/master/Makefile, for example.

Reply:
Also, add a blank line after this variable, like we do in other files, it improves readability.

Reply:
added, also updated to set `IMAGE_REGISTRY` instead of `IMAGE_TAG`